src/device-registry/utils/create-device.js

```diff
@@ -16,12 +16,17 @@ const logger = log4js.getLogger(
 );
 const qs = require("qs");
 const QRCode = require("qrcode");
-const { kafkaProducer } = require("../config/kafkajs");
 const httpStatus = require("http-status");
 let devicesModel = (tenant) => {
   return getModelByTenant(tenant, "device", DeviceSchema);
 };
 
+const { Kafka } = require("kafkajs");
+const kafka = new Kafka({
+  clientId: constants.KAFKA_CLIENT_ID,
```

Comment:
Addressing a runtime for Kafka producer where things were undefined.
So brought all the configurations into each respective util to prevent this cyclic dependency runtime error.

Repetition of code but prevent this bug from happening ever again, hopefully!

@BenjaminSsempala @NoahNsimbe